src/platform/nxp/common/DiagnosticDataProviderImpl.cpp

```diff
@@ -118,7 +118,7 @@ CHIP_ERROR DiagnosticDataProviderImpl::GetNetworkInterfaces(NetworkInterface **
     ifp->isOperational             = true;
     ifp->offPremiseServicesReachableIPv4.SetNull();
     ifp->offPremiseServicesReachableIPv6.SetNull();
-    ifp->type = EMBER_ZCL_INTERFACE_TYPE_ENUM_THREAD;
+    ifp->type = MATTER_ZCL_INTERFACE_TYPE_ENUM_THREAD;
```

Comment:
What is MATTER_ZCL_INTERFACE_TYPE_ENUM_THREAD?  It's not defined anywhere I can see....  It looks like this file was not correctly fixed when we moved to strong enums or something, _and_ isn't built?

Reply:
That is an excellent question, I don't see the EMBER_ZCL_INTERFACE_TYPE_ENUM_THREAD being declared anywhere inside the SDK so I'll most probably just remove the platform specific file from my PR unless @doru91 got something to say about this...

Reply:
removing from the PR